webhook/webhook.go

```diff
@@ -99,6 +99,9 @@ type ControllerOptions struct {
 	// TLS Client Authentication.
 	// The default value is tls.NoClientCert.
 	ClientAuth tls.ClientAuthType
+
+	// GroupName is the name of the group that is using this webhook.
+	GroupName string
```

Comment:
For serving, this is not a singular group.  Can this be extracted from the resource TypeMeta instead?

Reply:
e.g. we have `kmeta.OwnerRefable`, which most things should implement already, which should maybe be incorporated into either `HasSpec` or `GenericCRD`.

Reply:
Thinking about it, how about getting the group name from the `AdmissionRequest`?https://github.com/aliok/pkg/blob/139162bdb52ec4e177616188f27d1254d15b9bfb/webhook/webhook.go#L493 

I will give this a try.

`req.Resource.Group` is there.